app/src/main/res/drawable-v24/shield_dark.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
nit: license formatting.

We should maybe just fix this one day and just get the "This Source Code" start in the next line. Just putting this idea out there to see how it marinade. 

```
<?xml version="1.0" encoding="utf-8"?><!-- 
   - This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```